docs/resources/volumes.md

```diff
@@ -0,0 +1,61 @@
+---
+subcategory: "Unity Catalog"
+---
+# databricks_volumes (Resource)
+Volumes are a new Unity Catalog (UC) capability for accessing, storing, governing, and organizing files. Volumes unlock new processing capabilities for data governed by the Unity Catalog, including support for most machine learning and data science workloads. You can use volumes to store and access files in any format; data can be structured, semi-structured, or unstructured.
+
+With Volumes, files managed centrally in Unity Catalogare are organized under a 3-level namespace: `<catalog>.<schema>.<volume>`.
```

Comment:
Hi @bogdanrdc, I have used these from the doc, please let me know if these need to be changed or if you have any suggestions. Thank you.